### PR TITLE
chore: add script to setup test db

### PR DESCRIPTION
**Pull request by wa0x6e**
- Add a new `test:setup` yarn task to setup the test database. This task will run the commands to create the test database, and import the schema.
- Warn before booting the test suite is the test database is missing

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]
utAck

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "dev": "nodemon src/index.ts",
     "start": "node build/src/index.js",
     "start:test": "dotenv -e test/.env.test yarn dev",
+    "test:setup": "./test/setup.sh",
```
Comment (Todmy) [inline package.json:13]: what is the process of running it? Should I define some variables?
Screenshot 2023-10-10 at 11 56 45

Comment (wa0x6e) [reply package.json:13]: Is your mysql database running locally ? Do you have the `mysql` cli installed ?

Comment (wa0x6e) [reply package.json:13]: > yes, it runs locally. No, I don't have `mysql` cli - maybe we should add it to devDependencies if it's required?

The `mysql` cli should be shipped with the mysql install, and can not be installed with npm

**Review by Todmy — APPROVED** [review]
utACK

**Comment by wa0x6e** [comment]
Let me find another way to create the database

**Comment by Todmy** [comment]
> Let me find another way to create the database

I think it's not too critical. At least the script provides an example

**Comment by wa0x6e** [comment]
Script have been completely refactor to use 100% nodejs code, instead of shell command.

This will fix the `mysql` cli requirement issue.

**Review by Todmy — APPROVED** [review]
tACK

**Comment by Todmy** [comment]
thank you 🙏 it's awesome

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**